build.sh

```diff
@@ -0,0 +1,22 @@
+ECOV=0483b1e
+CWD=`pwd`
+
+if [ ! -d eco ]; then
+    git clone https://github.com/softdevteam/eco
+    cd eco
+    git checkout ${ECOV}
+    cd ..
+fi
+
+if [ -f results.csv ]; then
```

Comment:
I don't recommend deleting stuff ourselves, unless the user explicitly asked us to delete it.

Reply:
Hmm, but in that case, re-running the experiment is going to append the second run to the first, which I don't want.

Reply:
Yes, but at some point you're going to run `build.sh` and delete your old data, and then you'll be sad :) I don't suggest being very clever in this regard. In the error recovery experiment, I just `assert` that the file doesn't exist https://github.com/softdevteam/error_recovery_experiment/blob/master/runner/run.py#L17 -- if it does, it's the user's job to clear it up!

Reply:
Ah, that sounds fair.